Source/UI/KeyValue/KeyValuePaginatedTableViewDelegate.swift

```diff
@@ -38,6 +38,16 @@ class KeyValuePaginatedTableViewDelegate: NSObject, UITableViewDelegate {
     func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
         10
     }
+
+    func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        guard let dataSource = tableView.dataSource as? KeyValuePaginatedTableViewDataSource else {
+            return
+        }
+        guard let keyValue = dataSource.data.keyValueBy(index: indexPath.row) else {
+            return
+        }
+        Bots.current.markMessageAsRead(keyValue.key)
```

Comment:
I wonder if we should have some code similar to this for the notifications screen? I couldn't figure out how to clear the badge on the notifications tab. I think it would be fine if it cleared as soon as the cells are loaded there.

https://user-images.githubusercontent.com/1165004/179254481-9b33aaa0-5ba4-46af-9dba-6ef991054975.mp4